sdk/bazel-java-sdk/src/main/java/com/salesforce/bazel/sdk/bep/event/BEPBuildMetricsEvent.java

```diff
@@ -94,7 +113,7 @@ void parseDetails(JSONObject metricsDetail) {
     public String toString() {
         return "BEPBuildMetricsEvent [actionsCreated=" + actionsCreated + ", actionsExecuted=" + actionsExecuted
                 + ", usedHeapSizePostBuild=" + usedHeapSizePostBuild + ", cpuTimeInMs=" + cpuTimeInMs
-                + ", wallTimeInMs=" + wallTimeInMs + ", analysisPhaseTimeInMs=" + analysisPhaseTimeInMs + ", index="
+                + ", wallTimeInMs=" + wallTimeInMs + ", analysisPhaseTimeInMs=" + analysisPhaseTimeInMs + ", actionData=" + actionData + ", index="
```

Comment:
Currently this does not print each property of each action in the order as shown in the format above due iterating through the keys of the map not being in order of insertion. Not sure if I should make the print follow the order shown above?

Reply:
No, print order of attributes is irrelevant for toString(). The toString() is intended just for logging/pretty print on pages.